tianshou/env/worker/subproc.py

```diff
@@ -125,23 +126,29 @@ def _encode_obs(
 class SubprocEnvWorker(EnvWorker):
     """Subprocess worker used in SubprocVectorEnv and ShmemVectorEnv."""
 
-    def __init__(self, env_fn: Callable[[], gym.Env], share_memory: bool = False) -> None:
+    def __init__(
+        self,
+        env_fn: Callable[[], gym.Env],
+        share_memory: bool = False,
+        context: Literal["fork", "spawn"] | None = None,
+    ) -> None:
```

Comment:
Could you add a docstring explaining the semantics of fork, spawn and None? When would I want to use which? Would it make sense to also allow passing a context instance?

Reply:
In theory, one could also pass a context instance. Although I can't imagine a scenario where I would. Ultimately, it's a parameter that hardly ever needs to be touched, except something is not working :D

Reply:
Just to give an example
https://github.com/google-deepmind/mujoco/issues/742

Reply:
Then I would mention exactly that :)